src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  const testFunction = () => {};
```

Comment:
Would be good to use a valid props object here and populate it with the mock functions and test string so it can be passed into each of the test instances. Also make use of the `jest.fn()` mock